scala/lambdas/ingest-asset-opex-creator/src/main/scala/uk/gov/nationalarchives/ingestassetopexcreator/Lambda.scala

```diff
@@ -101,7 +104,7 @@ object Lambda {
 
   case class Input(id: UUID, batchId: String, executionName: String)
 
-  case class Config(dynamoTableName: String, dynamoGsiName: String, destinationBucket: String) derives ConfigReader
+  case class Config(dynamoTableName: String, dynamoGsiName: String, destinationBucket: String, roleArn: String) derives ConfigReader
```

Comment:
Should we call this `s3RoleArn`, maybe?

Reply:
I tried that, pure config tries to make me call it s-3-role-arn in the application config and I didn't want to convince it otherwise.